README.md

```diff
@@ -1,34 +1,28 @@
 # pm2-syslog
 
-Redirect all logs of PM2 + Apps managed into `/var/log/syslog`
+Redirect all logs of PM2 + Apps managed into `syslog`
 
-## Configure OS
-
-Edit `/etc/rsyslog.conf` and uncomment:
-
-```
-# provides UDP syslog reception
-module(load="imudp")
-input(type="imudp" port="514")
-```
-
-Restart rsyslog:
-
-```
-$ sudo service rsyslog restart
-```
 
 ## Install module
 
 ```
 # Install
-$ pm2 install pm2-syslog
+$ pm2 install pm2-syslog2
+
+# Configure pm2-syslog2 depending to where your syslog daemon listens
+$ pm2 set pm2-syslog:hostname localhost  # localhost is the default.
+$ pm2 set pm2-syslog:port 514  # 514 is the default.
+# or
+$ pm2 set pm2-syslog:path /dev/log
+
+# Optionally change the facility
+$ pm2 set pm2-syslog:facility local0  # user is the default
 
-# change the default hostname using
-$ pm2 set pm2-syslog:hostname localhost
```

Comment:
Didn't removed it, it's just a few line before. Also localhost is now the default in package.json.